R/tbl_survfit_quantiles.R

```diff
@@ -0,0 +1,245 @@
+#' Survival Quantiles
+#'
+#' Create a gtsummary table with Kaplan-Meier estimated survival quantiles.
+#'
+#' @param data (`data.frame`)\cr
+#'   A data frame
+#' @param y (`string`)\cr
+#'   A string with the survival outcome, e.g. `survival::Surv(time, status)`.
+#'   The default value is `"Surv_CNSR()"`, which is a shortcut for
+#'   `survival::Surv(time = AVAL, event = 1 - CNSR, type = "right", origin = 0)`.
+#' @param by ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   A single column from `data`. Summary statistics will be stratified by this variable.
+#'   Default is `NULL`, which returns results for the unstratified model.
+#' @param header (`string`)\cr
+#'   String for the header of the survival quantile chunks.
+#'   Default is `"Time to event"`.
+#' @param estimate_fun (`function`)\cr
+#'   Function used to round and format the estimates in the table.
+#'   Default is `label_style_number(digits = 1)`.
+#' @param method.args (named `list`)\cr
+#'   Named list of arguments that will be passed to `survival::survfit()`.
+#'
+#'   Note that this list may contain non-standard evaluation components, and
+#'   must be handled similarly to tidyselect inputs by using
+#'   rlang's embrace operator `{{ . }}` or `!!enquo()` when programming with this
+#'   function.
+#'
+#' @returns a gtsummary table
+#' @export
+#'
+#' @section ARD-first:
+#'
+#' This function is a helper for creating a common summary.
+#' But if you need to modify the appearance of this table, you may need to build
+#' it from ARDs.
+#'
+#' Here's the general outline for creating this table directly from ARDs.
+#' 1. Create an ARD of survival quantiles using `cardx::ard_survival_survfit()`.
+#' 2. Construct an ARD of the minimum and maximum survival times using `cards::ard_continuous()`.
+#' 3. Combine the ARDs and build summary table with `gtsummary::tbl_ard_summary()`.
+#'
+#' ```r
```

Comment:
maybe a problem of my machine(rstudio-server) but it does not show the syntax highlightings

Reply:
I don't think the help files support syntax highlighting unfortunately